MAINTAINERS.md

```diff
@@ -54,8 +54,9 @@ If you are an owner of the organization, you should be able to see an automated
 
 ## Organization members
 
-In addition everything that a repository collaborator can do,
-an organization member also has write access to all the repositories in the tldr-pages organization.
+In addition to everything that repository collaborators can do, organization members
+have write access to all the repositories in the tldr-pages organization,
+in addition to [a few extra maintenance tools](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/permission-levels-for-an-organization).
```

Comment:
I like this better.
```suggestion
and [a few extra maintenance tools](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/permission-levels-for-an-organization).
```

Reply:
This reads better as it does not repeat the wording.